test/helper.js

```diff
@@ -0,0 +1,133 @@
+'use strict';
+
+var WebdriverIO = require('../index.js'),
+    client, browserA, browserB;
+
+function end(passed, done) {
+    var SauceLabs, sessionID, endCommand;
+
+    if (process.env.TRAVIS_BUILD_NUMBER) {
+        SauceLabs = require('saucelabs');
+    }
+
+    if (!client) {
+        return done(false);
+    }
+
+    sessionID = (client.requestHandler || {}).sessionID;
+    endCommand = conf.runsWithSauce ? 'end' : 'endAll';
+
+    client[endCommand]().then(function() {
+        if (process.env.TRAVIS_BUILD_NUMBER) {
+            var sauceAccount = new SauceLabs({
+                username: process.env.SAUCE_USERNAME,
+                password: process.env.SAUCE_ACCESS_KEY
+            });
+
+            sauceAccount.updateJob(sessionID, {
+                passed: passed === true,
+                public: true
+            }, function(err, res) {
+                console.log(err || res);
+                if (err)
+                    return done(err);
+                return done();
+            });
+        } else {
+            return done();
+        }
+    }, done);
+}
+
+module.exports = {
+    noError: function(err) {
+        assert(err === undefined);
+    },
+    checkResult: function(expected) {
+        return function(result) {
+            if(expected instanceof Array) {
+                return expected.should.containDeep([result]);
+            }
+            expected.should.be.exactly(result);
+        };
+    },
+    setup: function(options) {
+        if(!options) {
+            options = {};
+        }
+
+        return function(done) {
+
+            function init() {
+                return WebdriverIO.remote(conf).init();
+            }
+
+            if(options.remoteOptions) {
+                conf = merge(conf, options.remoteOptions);
```

Comment:
`merge` is nowhere defined. Let's add `var merge = require('deepmerge');` to the top of the file